bindata/v4.0.0/service-serving-cert-signer-controller/clusterrole.yaml

```diff
@@ -36,9 +36,3 @@ rules:
   - watch
   - update
   - create
```

Comment:
this also should not need config map access.

also not sure if it needs delete on secrets.

Reply:
I don't think it needs create on services either: a84d0954